pkg/localmetrics/localmetrics.go

```diff
@@ -144,7 +146,18 @@ func NewMetricsCollector(store cache.Cache) *MetricsCollector {
 			ConstLabels: prometheus.Labels{"name": operatorName},
 			Buckets:     []float64{5, 10, 20, 30, 60, 120, 240, 300, 480, 600},
 		}),
-
+		accountClaimPendingDuration: prometheus.NewHistogram(prometheus.HistogramOpts{
+			Name:        "aws_account_operator_account_claim_pending_duration_seconds",
+			Help:        "The duration for account claim cr in pending mode",
+			ConstLabels: prometheus.Labels{"name": operatorName},
+			Buckets:     []float64{60, 120, 240, 300, 600},
+		}),
+		ccsAccountClaimPendingDuration: prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Discussion: I'm not sure if cardinality will be an issue, but can we include labels in these metrics for the specific account and how long it's taking?  Labels per account would include which shard we're on as well as the accountclaim name for troubleshooting purposes.  

If cardinality is an issue when including the labels we could potentially work around it, but I wonder if the metric can clear after it's been claimed.

For example, if there's an account that for whatever reason sits in pending, we'd get the alert at 10 minutes, but let's say it's just AWS API slowness that day and it clears up at 12 minutes.  Would the metric clear itself, or do we need to adjust the alert to ensure that it's only creating alerts when they're pending and the time pending is incrementing?  I'm not sure if that makes sense or not.